lib/nrf_modem_lib/lte_net_if/lte_net_if.c

```diff
@@ -302,29 +302,34 @@ static void pdn_event_handler(uint8_t cid, enum pdn_event event, int reason)
 
 static void lte_reg_handler(const struct lte_lc_evt *const evt)
 {
-	if (evt->type != LTE_LC_EVT_NW_REG_STATUS) {
-		return;
-	}
-
-	switch (evt->nw_reg_status) {
-	case LTE_LC_NW_REG_REGISTERED_HOME:
-		__fallthrough;
-	case LTE_LC_NW_REG_REGISTERED_ROAMING:
-		/* Mark serving cell as available. */
-		LOG_DBG("Registered to serving cell");
-		update_has_cell(true);
-		break;
-	case LTE_LC_NW_REG_SEARCHING:
-		/* Searching for a new cell, do not consider this cell loss unless it
-		 * fails (which will generate a new LTE_LC_EVT_NW_REG_STATUS event with
-		 * an unregistered status).
-		 */
-		break;
-	default:
-		LOG_DBG("Not registered to serving cell");
-		/* Mark the serving cell as lost. */
-		update_has_cell(false);
-		break;
+	if (evt->type == LTE_LC_EVT_MODEM_EVENT && evt->modem_evt == LTE_LC_MODEM_EVT_RESET_LOOP) {
+		LOG_WRN("The modem has detected a reset loop. LTE network attach is now "
+			"restricted for the next 30 minutes.");
+
+		LOG_DBG("For more information see the nRF91 AT Commands - Command "
+			"Reference Guide v2.0 - chpt. 5.36");
+
```

Comment:
I think this should be made more generic, because we now have separate AT command documentation for nRF9160 and nRF91x1. Also, it would be better not to refer to any specific document version.

How about something like this?
```suggestion
		LOG_DBG("For more information, see the AT command documentation for %MDMEV notification");
```